requirements.txt

```diff
@@ -9,3 +9,6 @@ pymc3==3.8
 python_dateutil==2.8.1
 scipy==1.4.1
 theano==1.0.4
+skimage==0.15.0
+semantic-version==2.8.5
+LDTk==1.1.1
```

Comment:
Packages needed for LDTk. skimage has to be 0.15.0 for now. Needs to be updated later to 0.17.2 as EXOTIC gets warning with the updated package.